packages/expo/src/executors/build/build.impl.ts

```diff
@@ -1,6 +1,9 @@
 import { ExecutorContext, names } from '@nx/devkit';
 import { resolve as pathResolve } from 'path';
 import { ChildProcess, fork } from 'child_process';
+import { removeSync } from 'fs-extra';
```

Comment:
Is this used you imported it but I don't see where it is used.

Reply:
removed